packages/account/src/Sections/Security/ConnectedApps/connected-apps-know-more.tsx

```diff
@@ -10,9 +10,8 @@ const openAPIManagingWebsite = () => {
     );
 };
 
-const ConnectedAppsArticle = () => (
-    <AccountArticle
-        className='connected-apps__article'
+const ConnectedAppsKnowMore = () => (
+    <Article
         title={localize('Want to know more about APIs?')}
```

Comment:
Can we change the type of title prop of Article to accept Component as well as string and pass <Localized/> ?